deployment/README.md

```diff
@@ -20,6 +20,18 @@ the project:
 gcloud auth application-default login
 ```
 
+Then, specify your Google Cloud project ID:
+```bash
+export GOOGLE_PROJECT={VALUE}
+```
+
+Eventually, customize the region (defaults to "us-east1"), and bucket name prefix
+(defaults to "tessera"):
+```bash
+export GOOGLE_REGION={VALUE}
```

Comment:
It feels a bit weird to me to have the behaviour of terragrunt be affected by environment vars (I think it's a niggle from "hermetic config as code & checked in" being violated), but:
a) I'm far from a TF/TG expert, and
b) I suppose this is meant to be an easily runnable example and having folks edit config files in a repo is a bit of an anti pattern.